assets/src/edit-story/components/inspector/design/useDesignPanels.js

```diff
@@ -29,9 +29,23 @@ import updateProperties from './updateProperties';
 
 function useDesignPanels() {
   const {
-    state: { selectedElementIds, selectedElements },
-    actions: { deleteSelectedElements, updateElementsById },
-  } = useStory();
+    selectedElementIds,
+    selectedElements,
+    deleteSelectedElements,
+    updateElementsById,
+  } = useStory(
+    ({
+      state: { selectedElementIds, selectedElements },
+      actions: { deleteSelectedElements, updateElementsById },
+    }) => {
+      return {
```

Comment:
nit: inline

Reply:
Strangely, this causes the argument list to be formatted on one line:

  } = useStory(
    ({ state: { selectedElementIds, selectedElements }, actions: { deleteSelectedElements, updateElementsById } }) => ({
      selectedElementIds,
      selectedElements,
      deleteSelectedElements,
      updateElementsById,
    });
  );